src/core/modules/firebase.ts

```diff
@@ -0,0 +1,21 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+export const firebaseConfig = {
+  apiKey: "AIzaSyDBYW8IrTR3_erHjdXFJxi83Sz8JGhmrmo",
```

Comment:
Create .env file in main directory and paste firebase config inside of it follow format like: 
`API_KEY="AIzaSyDBYW8IrTR3_erHjdXFJxi83Sz8JGhmrmo"`